### DISCUSSION TITLE
application中的印章识别示例，缺乏推理过程

### DISCUSSION THREAD

**Discussion by Intsigstephon** [PaddlePaddle/PaddleOCR#12281] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

**Comment by UserWangZz** [comment]
Thank you for your suggestion. We will continue to follow up and improve it
